Source/Core/Core/IOS/USB/Emulated/Skylanders/SkylanderFigure.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <Common/IOFile.h>
+
+namespace IOS::HLE::USB
+{
+struct SkylanderDateTime final
+{
+  u8 minute;
+  u8 hour;
+  u8 day;
+  u8 month;
+  u16 year;
+};
+
+struct SkylanderData final
+{
+  u16 money;
+  u16 heroLevel;
+  u32 playtime;
+  // Null-terminated UTF-16 string
+  std::array<u16, 0x10> nickname;
+  SkylanderDateTime lastReset;
+  SkylanderDateTime lastPlaced;
+};
+
+struct Trophydata final
+{
+  u8 unlockedVillains;
+};
```

Comment:
I don't know enough about Skylanders to know if there is more trophy data than this, but if `unlockedVillians` is going to be the only member of struct, consider removing this and change whatever uses this struct to be a `u8`.

Reply:
I'm not sure whether this is going to be the only data or if more data is stored on trophies. I figured that this would be more clear. Especially if I add more figure types later.

Reply:
Yeah single-member structs for type safety reasons is totally fine.

Reply:
Great!